recipe/meta.yaml

```diff
@@ -11,16 +11,27 @@ source:
     sha256: {{ sha256 }}
 
 build:
-  number: 2
+  number: 3
   skip: true  # [cuda_compiler_version in (undefined, "None")]
   script:
+    # For CUDA 11.0
+    - {{ PYTHON }} cupyx/tools/install_library.py --library cudnn --cuda 11.0 --prefix .  # [cuda_compiler_version == '11.0']
+    - export CUDA_PATH=/usr/local/cuda-11.0   # [cuda_compiler_version == '11.0']
+    - export NVCC="/usr/local/cuda-11.0/bin/nvcc -ccbin ${CXX}"  # [cuda_compiler_version == '11.0']
```

Comment:
`CUDA_PATH` is needed to guide the build system to locate CUDA 11's CUB headers. But if it's set, `nvcc` would complain `gcc` is not found (which is correct), so we also need to set `NVCC` to instruct `nvcc` to use conda's compiler.